torch/nn/quantized/modules/conv.py

```diff
@@ -92,11 +92,6 @@ def _save_to_state_dict(self, destination, prefix, keep_vars):
 
     @torch.jit.export
     def __getstate__(self):
```

Comment:
can we remove these now? since packed params are serializable

Reply:
I'd rather do that in a separate PR. This PR changes behavior in a ton of places and if it gets reverted it's better to have a minimal number of changes in here